<launch>

  <!-- Launch your obstacle publisher node -->
  <node pkg="greenship_pathfinder" type="dynamic_obstacle_simulator.py" name="dynamic_obstacle_simulation" output="screen" />

</launch>